nhrpd/nhrp_vty.c

```diff
@@ -1059,7 +1135,8 @@ static void interface_config_write_nhrp_map(struct nhrp_cache_config *c, void *d
 	vty_out(vty, " %s nhrp map %s %s\n", ctx->aficmd,
 		sockunion2str(&c->remote_addr, buf[0], sizeof(buf[0])),
 		c->type == NHRP_CACHE_LOCAL
-		? "local" : sockunion2str(&c->nbma, buf[1], sizeof(buf[1])));
+			? "local"
+			: sockunion2str(&c->nbma, buf[1], sizeof(buf[1])));
```

Comment:
same comment above, sockunion2str ==> %pSU

Reply:
Here the output can either be a fixed string ("local") or the sockunion contents. So %pSU won't work here.